package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
```

Comment:
My reason for putting a `test` line here at all was mostly as a joke/easter egg... I thought it was funny that our lint repo could pass its own linter :)